ignite/distributed/auto.py

```diff
@@ -234,7 +234,7 @@ def auto_model(model: nn.Module, sync_bn: bool = False, **kwargs: Any) -> nn.Mod
     return model
 
 
-def auto_optim(optimizer: Optimizer) -> Optimizer:
+def auto_optim(optimizer: Optimizer, **kwargs: Any) -> Optimizer:
```

Comment:
A tag `.. versionchanged:: 0.5.0` must be added in docstring. Please look the docstrings above for an example.